src/EditorFeatures/Core/Implementation/IntelliSense/Completion/Controller_Backspace.cs

```diff
@@ -63,10 +63,11 @@ private void ExecuteBackspaceOrDelete(ITextView textView, Action nextHandler, bo
                     this.TextView.Caret.PositionChanged += OnCaretPositionChanged;
                 }
 
-                var triggerInfo = CompletionTriggerInfo.CreateBackspaceTriggerInfo(deletedChar);
+                var trigger = new BackspaceOrDeleteCharCompletionTrigger(deletedChar, isBackspace: !isDelete, customTags: _completionTriggerTags);
```

Comment:
I would.  Frankly, I am somewhat surprised that we treat them the same.  (And, if we treat them the same, then why do we have to track if it was an actual delete vs a backspace?).  If they are treated differently somewhere, then we should def have different types IMO.